app/lib/theme/widgets/SelectWidget.tsx

```diff
@@ -0,0 +1,48 @@
+import { WidgetProps } from "@rjsf/core";
+import Dropdown from "@button-inc/bcgov-theme/Dropdown";
+import getRequiredLabel from "../utils/getRequiredLabel";
+
+interface Option {
+  type: string;
+  title: string;
+  enum: number[];
+  value: number;
+}
+
+const SelectWidget: React.FunctionComponent<WidgetProps> = (props) => {
+  const { id, onChange, schema, placeholder, label, required, uiSchema } =
+    props;
+
+  const options = schema.anyOf as Array<Option>;
```

Comment:
if we're relying on `anyOf` being used rather than `enum`/`enumNames`, we should probably check that it exists and throw an meaningful error here

Reply:
added a throw if anyOf is empty or undefined